packages/analytics/amplify_analytics_pinpoint_dart/lib/src/impl/analytics_client/event_client/event_client.dart

```diff
@@ -107,6 +107,8 @@ class EventClient implements Closeable {
     return completer.future;
   }
 
+  final _failedEventIds = <String, int>{};
```

Comment:
nit: the naming seems a bit confusing when I initially read this line, `_numOfFailuresByEvent`?